src/Model/Nosto/Entity/Helper/ProductHelper.php

```diff
@@ -62,7 +62,7 @@ public function reloadProduct(string $productId, SalesChannelContext $context):
         return $this->productRoute->load($productId, new Request(), $context, $criteria)->getProduct();
     }
 
-    private function getCommonCriteria(): Criteria
+    public function getCommonCriteria(): Criteria
```

Comment:
Didn't notice it was a private function. Would it makes sense to move `getShopwareProducts()` to the helper?

Reply:
Alright, I'll move the `getShopwareProducts` method to the helper.